### ISSUE TITLE
Partially match in templates

### ISSUE THREAD

**Issue by ifuyivara** [SpectoLabs/hoverfly#242]
With the addition of template partial matching I'm able to create templates for my requests. However I couldn't find in the docs how to create a template match for a particular path.

Let's say I have a path that looks like this:
```
GET
/user/2312323/status
```
How can I match any user id on my template? I was expecting to be able to type something similar to this but it didn't work:
```
"requestTemplate": {
            "path": "/user/*/status",
            "method": "GET",
            "scheme": "http"
        },
```

Thanks!
-Ivan

**Comment by benjih** [comment]
Hey @ifuyivara,

I've encountered a similar problem to this myself. Currently, the templates do not support either wildcarding or regex. They currently match as is with the request, making it impossible to define one template for viewing multiple users.

This is definitely on our roadmap and I will be looking into adding this feature soon.

**Comment by benjih** [comment]
Hey @ifuyivara,

This feature was added as part of #256. This feature is now available in [v0.8.2](https://github.com/SpectoLabs/hoverfly/releases/tag/v0.8.2) our most recent release of Hoverfly. I hope this helps.

If you have any issues with this feature, please let us know.
